src/ducks/categories/CategoriesPage.jsx

```diff
@@ -47,7 +56,7 @@ class CategoriesPage extends Component {
     } = this.props
     const isFetching =
       isCollectionLoading(transactions) || isCollectionLoading(settings)
-    const { showIncomeCategory } = (settings.data && settings.data[0]) || false
+    const { showIncomeCategory } = this.getSettings()
```

Comment:
Nit: since we need to retrieve the settings in other places, I think we should remove `getSettings` and just to `getSettingsFromCollection(this.props.settingsCollection)` where we need it.